CMakeLists.txt

```diff
@@ -39,6 +39,19 @@ target_include_directories(thread_priority PUBLIC
 ament_target_dependencies(thread_priority PUBLIC ${THIS_PACKAGE_INCLUDE_DEPENDS})
 target_link_libraries(thread_priority PUBLIC cap)
 
+add_library(realtime_helpers SHARED
+  src/realtime_helpers.cpp
+)
+target_compile_features(realtime_helpers PUBLIC cxx_std_17)
+target_include_directories(realtime_helpers PUBLIC
+  $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
+  $<INSTALL_INTERFACE:include/realtime_tools>
+)
+ament_target_dependencies(realtime_helpers PUBLIC ${THIS_PACKAGE_INCLUDE_DEPENDS})
+if(UNIX)
+  target_link_libraries(realtime_helpers PUBLIC cap)
+endif()
```

Comment:
@fmauch @christophfroehlich Do you think it is better to create a new library out of it or let's add it to the main library `realtime_tools` one

Reply:
As you prefer..

Reply:
Done. Thanks